core/string_db.h

```diff
@@ -113,6 +113,9 @@ class StringName {
 		else
 			return 0;
 	}
+	_FORCE_INLINE_ const void *ptr_hash() const {
```

Comment:
I'm actually not sure about the naming here. Also @reduz might want to let us know if he thinks this is a good idea or not.